test/data/03/README.md

```diff
@@ -0,0 +1,21 @@
+# Model 3 ~ Simple model with quadratic terms and metadata
+
+$$
+\left\lbrace
+\begin{align*}
+f(x_1, x_3, x_5) = ~? \\[1ex]
+H(s_1, s_3, s_5) = ~?
+\end{align*}
+\right.
+$$
+
+## Status Table
+
+| Format   | Domain | Status |
+| :------- | :----: | :----: |
+| BQPJSON  |  Bool  |   ❌    |
+| BQPJSON  |  Spin  |   ❌    |
+| MiniZinc |  Bool  |   ❌    |
+| MiniZinc |  Spin  |   ❌    |
+| QUBO     |  Bool  |   ❌    |
+| Qubist   |  Spin  |   ❌    |
```

Comment:
What's the motivation in this example?

Reply:
Only examples 0-2 are ready, not the other ones, sorry